[2023-10-16T15:12:20.217+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: execute_sqlite_pipeline.insert_data manual__2023-10-16T14:12:16.465551+00:00 [queued]>
[2023-10-16T15:12:20.219+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: execute_sqlite_pipeline.insert_data manual__2023-10-16T14:12:16.465551+00:00 [queued]>
[2023-10-16T15:12:20.219+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-16T15:12:20.222+0100] {taskinstance.py:1382} INFO - Executing <Task(SqliteOperator): insert_data> on 2023-10-16 14:12:16.465551+00:00
[2023-10-16T15:12:20.224+0100] {standard_task_runner.py:57} INFO - Started process 99012 to run task
[2023-10-16T15:12:20.227+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'execute_sqlite_pipeline', 'insert_data', 'manual__2023-10-16T14:12:16.465551+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/sqlite_pipeline.py', '--cfg-path', '/var/folders/_3/bj8r7kz51nz4g7hl_dpr6jdw0000gn/T/tmp8og2kbhq']
[2023-10-16T15:12:20.230+0100] {standard_task_runner.py:85} INFO - Job 94: Subtask insert_data
[2023-10-16T15:12:20.256+0100] {task_command.py:416} INFO - Running <TaskInstance: execute_sqlite_pipeline.insert_data manual__2023-10-16T14:12:16.465551+00:00 [running]> on host 192.168.1.112
[2023-10-16T15:12:20.274+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='workflow' AIRFLOW_CTX_DAG_ID='execute_sqlite_pipeline' AIRFLOW_CTX_TASK_ID='insert_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-16T14:12:16.465551+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-16T14:12:16.465551+00:00'
[2023-10-16T15:12:20.275+0100] {sql.py:274} INFO - Executing: 
            INSERT INTO new_users (name, age, is_active) VALUES
            ('Bob', 23, true),
            ('Alice', 25, true),
            ('John', 27, false),
            ('Jane', 21, true);
        
[2023-10-16T15:12:20.276+0100] {base.py:73} INFO - Using connection ID 'db_sqlite_conn' for task execution.
[2023-10-16T15:12:20.310+0100] {base.py:73} INFO - Using connection ID 'db_sqlite_conn' for task execution.
[2023-10-16T15:12:20.310+0100] {sql.py:418} INFO - Running statement: 
            INSERT INTO new_users (name, age, is_active) VALUES
            ('Bob', 23, true),
            ('Alice', 25, true),
            ('John', 27, false),
            ('Jane', 21, true);
        , parameters: None
[2023-10-16T15:12:20.311+0100] {sql.py:427} INFO - Rows affected: 4
[2023-10-16T15:12:20.313+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=execute_sqlite_pipeline, task_id=insert_data, execution_date=20231016T141216, start_date=20231016T141220, end_date=20231016T141220
[2023-10-16T15:12:20.329+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-16T15:12:20.336+0100] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
